lib/wasmer-types/Cargo.toml

```diff
@@ -22,3 +22,6 @@ default = ["std", "enable-serde"]
 std = ["serde/std"]
 core = []
 enable-serde = ["serde", "cranelift-entity/enable-serde"]
+
+# experimental / in-development features
+experimental-reference-types-extern-ref = []
```

Comment:
I'd name it `reference-types` directly. It's experimental per se, but it's unlikely to have another implementation (all implementations wil be experimental), a feature can be also experimental per se.
Therefore simplifying the feature name seems like a good thing to do!

Reply:
That's good feedback but I think the name `reference-types` would be misleading because this feature only concerns the `ExternRef` API in Wasmer. Reference types are always enabled and can't easily be disabled, statically in the code I mean.

The runtime config that allows Wasm modules to be compiled with reference types has it disabled by default. So users must update that config option or pass the `--enable-reference-types` flag to enable it wit the Wasmer CLI currently.

Happy to change the name of the feature flag though